EpsilonLights/Src/Lights.c

```diff
@@ -177,29 +182,34 @@ void updateStrobeLight(void const* arg)
 
     for (;;)
     {
-        osDelayUntil(&prevWakeTime, LIGHTS_UPDATE_FREQ);
-        strobeLight = (auxBmsInputs[1] >> 0) & STROBE_FAULT_MASK;
+        updateStrobeLight2(&prevWakeTime, &blinkerTimer, &strobeLight);
```

Comment:
I think if you make the `updateStrobeLight` to be `updateStrobeLightTask` and then your function to be `updateStrobLight` it would be better

Reply:
this would require us to make changes main.c, if thats ok